### PR TITLE
Laravel 5.5 Support

### PR DESCRIPTION
**Pull request by gtapps**
Hello,

This pull request includes support for laravel 5.5.

Cheers,
Gabriel

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 language: php
 
 php:
-  - 5.5
   - 5.6
```
Comment (jarektkaczyk) [inline .travis.yml]: remove 5.6 as well

File: src/Hookable.php [hunk 2]
```diff
@@ -122,15 +122,15 @@ public function setAttribute($key, $value)
      */
     public function save(array $options = [])
     {
-        if (!parent::save($options)) {
+        if (!($saved = parent::save($options)) && $this->isDirty()) {
```
Comment (jarektkaczyk) [inline src/Hookable.php]: why this?

File: src/Hookable.php [hunk 3]
```diff
@@ -142,7 +142,7 @@ public function save(array $options = [])
      * @param null $attributes
      * @return bool
      */
-    public function isDirty($attributes = null)
+    /*public function isDirty($attributes = null)
```
Comment (jarektkaczyk) [inline src/Hookable.php]: why commented out?

**Comment by ragingdave** [comment]
@gtapps Are you planning on finishing this one  or not too?

**Comment by ragingdave** [comment]
@gtapps ?

**Pull request closed without merging**